### PR TITLE
update attribution recipe

### PR DESCRIPTION
**Pull request by architgoyal1**

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

**Comment by igneel64** [comment]
Hey @architgoyal1 , not sure if this is in scope, but since we are updating the attribution recipe, does it make sense to promote the [marketing attribution accelerator](https://snowplow.io/data-product-accelerators/fractional-attribution/) as we do with the ecommerce recipe page ?
image

**Comment by jrpeck1989** [comment]
Not sure if it's possible to address, but the images of the different attribution models have black text, which isn't viewable when the docs are in dark more

**Comment by jrpeck1989** [comment]
I think the steps seem a bit weird and difficult to follow. I don't think "Step 1" should be "look at this demo". I think that should be separate, then followed by the steps to actually implement the solution. So:

* What you will achieve - demo dataset, and charts

1. Implement tracking through GTM 
2. Run `CREATE TABLE` statement and then select from it
3. Attach table to a free BI tool (Looker Studio or whatever)

File: docs/recipes/recipe-marketing-attribution/images/datamodel.png
Comment (rlh1994) [inline docs/recipes/recipe-marketing-attribution/images/datamodel.png:1]: Were these generated via drawio? If so could you save them as an embedded diagram so it's possible to edit these in the future if needed?

File: docs/recipes/recipe-marketing-attribution/index.md [hunk 1]
```diff
@@ -225,25 +193,132 @@ FROM derived.marketing_attribution
...
+Now you just need to connect to your warehouse and starting building up an asset of customer data that you can get better and better attribution insights out of over time!
+
+To connect to your warehouse / lake, you need to go into the Snowplow UI, click 'Upgrade Now' and follow the setup wizard for BDP Cloud (you will need a credit card for this), this is a production-ready platform that can reliably load live events to your warehouse / lake.
+
+![](images/destination.png)
+
+When your connection is confirmed, deploy our prebuilt dbt packages following our quickstart guides:
+
+1. Snowplow [web package](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/web/)
+2. Snowplow [attribution package](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/)
```
Comment (rlh1994) [inline docs/recipes/recipe-marketing-attribution/index.md:290]: How do Try customers deploy models, can they do it via BDP or would they have to run and manage these locally?

File: docs/recipes/recipe-marketing-attribution/index.md [hunk 2]
```diff
@@ -225,25 +193,132 @@ FROM derived.marketing_attribution
...
+
+![](images/datamodel.png)
+
```
Comment (rlh1994) [inline docs/recipes/recipe-marketing-attribution/index.md:293]: This section in general feels a little thrown in here I feel, the image feels a little disconnected from the text and it's not explained what deploying these packages will actually have the user achieve/output by the end of it?

File: docs/recipes/recipe-marketing-attribution/index.md [hunk 3]
```diff
@@ -225,25 +193,132 @@ FROM derived.marketing_attribution
...
+Do this by copying the endpoint from the [Snowplow BDP UI](https://console.snowplowanalytics.com/environments) into your tracking code in GTM:
+
+```js
+<script type="text/javascript" async=1>
+...
+
+snowplow("newTracker", "sp", "com-acme-saas1.collector.snplow.net", {
+...
+```
+:::
```
Comment (rlh1994) [inline docs/recipes/recipe-marketing-attribution/index.md:306]: This note would make more sense before the instructions to add the dbt models I think

File: docs/recipes/recipe-marketing-attribution/index.md [hunk 4]
```diff
@@ -225,25 +193,132 @@ FROM derived.marketing_attribution
...
-- With the out of the box tracking, you have automatically captured the marketing and referrer information for each page view and session.
-- By instrumenting a custom conversion event, you are collecting all the data you need to get started with attribution.
-- By running a simple query that attributes the conversions to the marketing channels based on three popular attribution models, you can explore how Snowplow enables you to take control of your marketing attribution.
+#### [Optional] For more confidence, track a dedicated conversion event
 
-## What you might want to do next
+<details>
+  <summary>
+Design and implement a `conversion` event
+  </summary>
```
Comment (rlh1994) [inline docs/recipes/recipe-marketing-attribution/index.md:224]: Backticks don't work in `summary` tags, you'll need to swap to a `<code>` tag around it

File: docs/recipes/recipe-marketing-attribution/index.md [hunk 5]
```diff
@@ -5,113 +5,79 @@ date: "2020-10-12"
...
+## What you'll be doing
+The most important goal of this guide is to help you learn something new about your customers!
 
-#### Designing the `conversion` event
+:::note
+Prerequisites: None!
 
-We have already created a custom `conversion` event for you in [Iglu Central](http://iglucentral.com/).
+You don't need any tooling to build a v1 model, you will need a warehouse and dbt to do the more advanced final section of this guide.
+:::
```
Comment (rlh1994) [inline docs/recipes/recipe-marketing-attribution/index.md:30]: This feels a little disingenuous as the requirements increase throughout the recipe, for step 3 you need to be a BDP Cloud customer. Could we call out the change between step 2 and step 3, or be really clear that you can see the output and everything by just getting to step 2 which can all be done with Try Snowplow. My worry is if I skim a tutorial and see I need a credit card for a step, I might not try the tutorial at all.

File: docs/recipes/recipe-marketing-attribution/images/datamodel.png
Comment (architgoyal1) [reply docs/recipes/recipe-marketing-attribution/images/datamodel.png:1]: I have updated this diagram for now and used an existing one marketing had lying around

Comment (architgoyal1) [reply docs/recipes/recipe-marketing-attribution/images/datamodel.png:1]: Neither the old or new one were made in drawio

File: docs/recipes/recipe-marketing-attribution/index.md (hunk 2 above)
Comment (architgoyal1) [reply docs/recipes/recipe-marketing-attribution/index.md:293]: Agreed, I rushed this last bit - I have tried to add to it in my latest commit

File: docs/recipes/recipe-marketing-attribution/index.md (hunk 3 above)
Comment (architgoyal1) [reply docs/recipes/recipe-marketing-attribution/index.md:306]: Done

File: docs/recipes/recipe-marketing-attribution/index.md (hunk 5 above)
Comment (architgoyal1) [reply docs/recipes/recipe-marketing-attribution/index.md:30]: Thats fair, I have tried to distinguish the two now

File: docs/recipes/recipe-marketing-attribution/index.md (hunk 4 above)
Comment (architgoyal1) [reply docs/recipes/recipe-marketing-attribution/index.md:224]: Updated

**Comment by architgoyal1** [comment]
> I think the steps seem a bit weird and difficult to follow.

@jrpeck1989 the steps are indeed a bit odd - the reasoning is that I want to use this page in a marketing campaign aimed at practitoners with the goal of getting them to sign up to BDP Cloud. Unfortunately, they need to first sign up to Try Snowplow and then upgrade to Cloud which is why I was trying to make a logical journey through both products.

I have tried to reword the section titles to make it less jarring - let me know if its passable, if not Ill try and pull it out into a separate section.

**Review by jrpeck1989 — APPROVED** [review]
All good for now

**Pull request merged**